Content.Shared/Chat/SharedChatSystem.cs

```diff
@@ -90,7 +90,7 @@ public SpeechVerbPrototype GetSpeechVerb(EntityUid source, string message, Speec
     /// <remarks>
     /// This is primarily for the chat emote sanitizer, which can match against ":b" as an emote, which is a valid radio keycode.
     /// </remarks>
-    public void GetRadioKeycodePrefix(EntityUid source,
+    public void SplitRadioKeycodePrefix(EntityUid source,
```

Comment:
No need to change the function name; its an unnecessary breaking change

Reply:
I was originally planning to change the name for the old PR but forgot to push...

I pity anyone who has or will have to depend on this code.

Name change reverted.